### ISSUE TITLE
How do I know the status of a command?

### ISSUE THREAD

**Issue by itaibh** [MicrosoftDocs/azure-devops-docs#11014]
When running a command I can't see (like file attachment), how can I know if it was successful or not?
I have a customer using an extension I wrote which uses attachments, and it stopped working at some scenarios, and I can't figure out why. I can tell that there are no attachments in those cases since the server side which looks for them returns an empty list in the callback, which means either the attachment wasn't successful for some reason or the reading wasn't successful for some reason.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 16bf0bbb-8b63-6384-aa33-b5b1e2b8cfe3
* Version Independent ID: f4da8dbd-a02f-bbdf-2cf2-60e081339237
* Content: [Logging commands - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=bash#addattachment-attach-a-file-to-the-build)
* Content Source: [docs/pipelines/scripts/logging-commands.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/master/docs/pipelines/scripts/logging-commands.md)
* Product: **devops**
* Technology: **devops-cicd-scripts**
* GitHub Login: @juliakm
* Microsoft Alias: **jukullam**

**Comment by TimShererWithAquent** [comment]
@itaibh Thanks for your question. This scenario sounds like something that we should document. Not sure if this is the right page, but we'll look into it. 

In the meantime, consider these resources:

- [Azure DevOps Services](https://azure.microsoft.com/support/devops/) 
- [Azure DevOps Support Bot](https://azuredevopsvirtualagent.azurewebsites.net/) 
- [Azure DevOps on Stack Overflow](https://stackoverflow.com/questions/tagged/azure-devops)

**Comment by itaibh** [comment]
I posted the question in StackOverflow (https://stackoverflow.com/questions/68242950/how-do-i-know-the-status-of-a-logging-command), so far no responses. 

I tried the support bot but I couldn't find the correct phrase and it kept on answering with "Can you rephrase?" and in the options I couldn't find anything relevant.

I tried asking in the community page on Facebook, linking to the StackOverflow question, no luck there either.

Please can anyone help me solve this?

**Comment by juliakm** [comment]
@itaibh Thanks for the question. Tasks are designed to be lightweight and do not include feedback by design. Our product team recommends you find an equivalent task that you can use. They suggested looking at the [Copy Files task](https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/copy-files?view=azure-devops&tabs=yaml).

**Comment by itaibh** [comment]
Dear @juliakm,
Thank you for your comment.
It's not been helpful at all and the fact that you closed this ticket right after referring me to something completely unrelated to what I was trying to do tells me all I need to know about the level of support I can get from Microsoft on this channel.
Good day.
